docs/ListTutorial.md

```diff
@@ -515,15 +516,22 @@ To use `<Datagrid>` on desktop and `<SimpleList>` on mobile, use the `useMediaQu
 ```jsx
 // in src/posts.js
 import * as React from 'react';
-import { useMediaQuery } from '@mui/material';
+import { useMediaQuery, Theme } from '@mui/material';
 import { List, SimpleList, Datagrid, TextField, ReferenceField } from 'react-admin';
 
+type Post = {
+    id: number;
+    title: string;
+    views: number;
+    published_at: string;
+}
+
 export const PostList = () => {
-    const isSmall = useMediaQuery(theme => theme.breakpoints.down('sm'));
+    const isSmall = useMediaQuery<Theme>(theme => theme.breakpoints.down('sm'));
     return (
         <List>
             {isSmall ? (
-                <SimpleList
+                <SimpleList<Post>
```

Comment:
Is it for comfort, or does SimpleList require a type to compile?

Reply:
You may have an error about accessing an `any` otherwise, depending on your TS config